ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/FontAwesome.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.ide.ui;
+package org.eclipse.che.ide.api;
```

Comment:
I think **che-core-ide-ui** module is the right place for that. And this class isn't a part of IDE API.